.bazelrc

```diff
@@ -20,3 +20,13 @@ build --deleted_packages tests/ios/frameworks/sources-with-prebuilt-binaries
 # Enable dbg compilation mode in this repo, so we can test xcodeproj-built
 # binaries contain debug symbol tables.
 build --compilation_mode=dbg
+
+# Use 'worker' strategy for swift compilation
+build --strategy=SwiftCompile=worker
+
+# This flips index_while_building_v2 on
+build --features swift.index_while_building_v2
+
+# Default WMO enabled. As a followup rules_ios should have variant with and
+# without this enabled.
+build --swiftcopt -whole-module-optimization
```

Comment:
what is the reason for enabling whole-module-optimization? Was this to address another bug that was exhibited?

Reply:
Yeah we use WMO as default, need to send a patch for the non WMO case. ATM if you set `iwbv2` then you've gotta use WMO until we fix that one

Reply:
@amberdixon fixed here - moves it to conditionally use the non transient output for non WMO: https://github.com/bazelbuild/rules_swift/pull/567/commits/ceaacfc34dba1575e8555db50ddb4009772bf9c8 - I can bump the sha of this.